library/std/src/backtrace.rs

```diff
@@ -117,15 +112,19 @@ pub struct Backtrace {
 /// whether it is empty for some other reason.
 #[non_exhaustive]
 #[derive(Debug, PartialEq, Eq)]
```

Comment:
```suggestion
#[derive(Clone, Debug, PartialEq, Eq)]
```

?

Reply:
That seems pretty reasonable to me :+1: I can't imagine this `enum` acquiring any fields, let alone any non-`Clone` ones